plugins/woocommerce/src/StoreApi/SessionHandler.php

```diff
@@ -78,13 +79,14 @@ public function get_session( $customer_id, $default = false ) {
 
 		$value = $wpdb->get_var(
 			$wpdb->prepare(
-				"SELECT session_value FROM $this->table WHERE session_key = %s", // phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+				'SELECT session_value FROM %i WHERE session_key = %s',
```

Comment:
[TIL](https://developer.wordpress.org/reference/classes/wpdb/prepare/#changelog)

Reply:
@straku just pointing out this is cool, it could have been used on Points and rewards 👀